website/content/en/preview/tasks/scheduling.md

```diff
@@ -447,10 +449,13 @@ spec:
   - key: kubernetes.io/arch
     operator: In
     values: ["arm64"]
+  - key: node.kubernetes.io/instance-type
+    operator: In
+    values: ["c4.large", "c4.xlarge"]
 ```
 
 {{% alert title="Note" color="primary" %}}
-Based on the way that Karpenter does pod batching and bin packing, it is not guaranteed that Karpenter will always choose the highest priority provisioner given specific requirements. For example, it is possible that a pod that has a node selector on a an `arm64` architecture may be provisioned first and all other workload pods from that batch may also schedule on that node. The behavior may also occur if existing capacity is available, as the kube-scheduler will schedule the pods instead of allowing Karpenter to provision a new node.
+Based on the way that Karpenter does pod batching and bin packing, it is not guaranteed that Karpenter will always choose the highest priority provisioner given specific requirements. For example, it is possible that a pod that has a node selector on a an `arm64` architecture to be provisioned, causing all other workload pods from that batch to also schedule on that node (even if those pods had no explicit requirement for `arm64` architecture). The behavior may also occur if existing capacity is available, as the kube-scheduler will schedule the pods instead of allowing Karpenter to provision a new node.
```

Comment:
```suggestion
Based on the way that Karpenter performs pod batching and bin packing, it is not guaranteed that Karpenter will always choose the highest priority provisioner given specific requirements. For example, it is possible that a pod that has a node selector on an `arm64` architecture to be provisioned, causing all other workload pods from that batch to also schedule on that node (even if those pods had no explicit requirement for `arm64` architecture). The behavior may also occur if existing capacity is available, as the kube-scheduler will schedule the pods instead of allowing Karpenter to provision a new node.
```